[google_maps_flutter] Fix various bugs

seungsoo47 opened the pull request:
* Remove analysis_options.yaml.
* google_maps_flutter version up.
* Fix crash issues and bugs.

Signed-off-by: leess <seungsoo47.lee@samsung.com>

seungsoo47 commented:
This patch should be merged after the latest engine/webview plugin is released. It is because the interfaces of the two modules do not match.

seungsoo47 commented:
https://github.com/flutter-tizen/plugins/issues/389

HakkyuKim commented:
Since `0.1.2` is not released, all changes can be part of `0.1.2` and not `0.1.3`. Do you have a reason to release two versions here?

seungsoo47 commented:
> Since `0.1.2` is not released, all changes can be part of `0.1.2` and not `0.1.3`. Do you have a reason to release two versions here?

I thought that @bbrto21 already released `0.1.2` in the below patch.
https://github.com/flutter-tizen/plugins/pull/369/files
However, it just seems the version was not written in Changelog.md.

HakkyuKim commented:
Ah, I guess that was supposed to be released with CHANGELOG, since it wasn't, we can release `0.1.2` together with the current changes.

seungsoo47 commented:
> Ah, I guess that was supposed to be released with CHANGELOG, since it wasn't, we can release `0.1.2` together with the current changes.

Thank you very much :)

bbrto21 commented:
> * google_maps_flutter version up.

Thanks Thank you for your hard work. but I have a question.
Shouldn't we update the example to the latest one instead of resolving issues of  lint that exists in current example?
If not, do you have a plan to update the example with implementing additional feature of this plugin in the future?

bbrto21 commented:
I'm asking that if it's necessary to copy the latest original example to this plug-in.
I tried it based on this PR, and it seems to be no problem with the execution and behavior of the example, but some lint issues have been reported after the analyze.

seungsoo47 commented:
> I'm asking that if it's necessary to copy the latest original example to this plug-in. I tried it based on this PR(I just replaced lib dir of the example.), and it seems to be no problem with the execution and behavior of the example, but some lint issues have been reported after the analyze.

Thank you for your kind review. 😀 I am not sure that you remember what I said for this plugin. As you know, this plugin is based on webview plugin and uses google map web apis. However, the original examples are based on google map apis and the apis are different from web version apis a little bit. So, I removed some examples related to unsupported apis. By the way, if I miss some examples based on supported apis, please let me know it. ^^

HakkyuKim commented on packages/google_maps_flutter/CHANGELOG.md:
```diff
@@ -1,3 +1,12 @@
+## 0.1.3
+
+* Fix variouse bugs.
+* Update webview_flutter_tizen to 0.5.1.
+
+## 0.1.2
+
+* Update dependency package.
+
```
Is the version related changes in progress? 

```suggestion
## 0.1.2

* Fix variouse bugs.
* Update webview_flutter to 3.0.4.
* Update webview_flutter_tizen to 0.5.1.

```

And all version related changes in `pubspec.yaml`s and `README`s.

seungsoo47 replied:
Oh, I am sorry for being too late. I fixed them. Thank you 😀

HakkyuKim commented on packages/google_maps_flutter/CHANGELOG.md:
```diff
@@ -1,6 +1,9 @@
 * Fix variouse bugs.
+* Update webview_flutter to 3.0.4.
+* Update webview_flutter_tizen to 0.5.1.
+* Update dependency package.
```
Thanks for the changes, I'm not sure if there are other meanings to "Update dependency package" other than bumping up `webview_flutter` and `webview_flutter_tizen`. This line is not needed if you meant those two packages.

seungsoo47 replied:
Ah~ As I mentioned above, actually @bbrto21 's patch was 0.1.2 and it will be released with this PR. So I thought I needed `Update dependency package` about @bbrto21 patch. What do you think? :)

bbrto21 replied:
@seungsoo47 I think you don't need a last line. 
```suggestion
```

swift-kim commented:
Any updates? This package is the last package that depends on the `analysis_options_legacy.yaml` file. The file will be deleted after this PR is merged.

seungsoo47 commented:
> Any updates? This package is the last package that depends on the `analysis_options_legacy.yaml` file. The file will be deleted after this PR is merged.
No more updates. Please merge this pr.
